nltk/tokenize/casual.py

```diff
@@ -262,16 +270,34 @@ def tokenize(self, text):
                               x.lower()), words))
         return words
 
+######################################################################
+# Normalization Functions
+######################################################################
+
+def reduce_lengthening(text):
+    '''
+    Replace character sequences of length 3 or greater with sequences of length 3.
```

Comment:
Also, why not extend this to 2-character repetitions like `hahahaha` and `lolololol`?